app/src/test/java/io/gnosis/safe/ui/transactions/TransactionListViewModelTest.kt

```diff
@@ -489,7 +489,8 @@ class TransactionListViewModelTest {
                 confirmationsTextColor = R.color.primary,
                 threshold = 2,
                 confirmations = 2,
-                addressInfo = AddressInfoData.Default
+                addressInfo = AddressInfoData.Default,
+                isCancellation = false
```

Comment:
Just small comment here, 

As we agreed later on there will be new type of transaction for rejection transaction, and in UI it's handled as new type so I recommend to create new model type instead of just flag.